lib/config/guiconfiguration.go

```diff
@@ -7,19 +7,23 @@
 package config
 
 import (
+	"encoding/base64"
 	"net/url"
 	"os"
 	"regexp"
 	"strconv"
 	"strings"
 
-	"golang.org/x/crypto/bcrypt"
-
 	"github.com/syncthing/syncthing/lib/rand"
+	"golang.org/x/crypto/bcrypt"
```

Comment:
Oh, I assumed this was one of the things gofmt would be taking care of. Will fix.

Reply:
Fixed in d20819fabfacbbcf961863c0346cb74610c4b346. I assume the imports in `.pb.go` files are exempt since those files are wholly generated.

Reply:
Yeah go fmt is a weird case in my opinion: It's more or less universally used, with the accompanied love-hate attitude to it, but it's not very opinionated. It does fix/determine a bunch of aspects of formatting, but also leaves a lot of aspects open/undefined.

Reply:
> Fixed in [d20819f](https://github.com/syncthing/syncthing/commit/d20819fabfacbbcf961863c0346cb74610c4b346). I assume the imports in `.pb.go` files are exempt since those files are wholly generated.

Ups, didn't see this one yet when posting my previous comment:  
Yeah I was only commenting because it was a formatting only change in your PR. Autogenerated files clearly are excempt, and if it would have been a new file I probably wouldn't even have noticed :)